### DISCUSSION TITLE
围棋AI串烧（AlphaGo、AlphaGo zero、kataGo）

### DISCUSSION THREAD

**Discussion by absolutemind** [mli/paper-reading#119] [Ideas]
AlphaGo
Mastering the game of Go with deep neural networks and tree search

AlphaGo Zero
Mastering  the game of go without human knowledge

kataGo
Accelerating Self-Play Learning in Go

**Comment by BIGBALLON** [comment]
提供一个很久以前的学习资料  （各位远程组会朋友们可以随意参考）

- https://github.com/BIGBALLON/Toward-AGZ
